### ISSUE TITLE
2 Elements using the same ID

### ISSUE THREAD

**Issue by SamBrishes** [RatMD/bloghub-plugin#5]
Chrome Warning: 

```
[DOM] Found 2 elements with non-unique id #bloghubCommentSection6368b0bd2e5b6_username: (More info: https://goo.gl/9p2vKq) 
<input id=​"bloghubCommentSection6368b0bd2e5b6_username" type=​"text" name=​"comment_name" value=​"Sam" placeholder=​"Your Username" class=​"form-control" readonly>​
<input id=​"bloghubCommentSection6368b0bd2e5b6_username" type=​"text" name=​"comment_user" autocomplete=​"off" value placeholder=​"Your Username" class=​"form-control from-control-bloghub-special-assignment">​
```

**Comment by SamBrishes** [comment]
Fixed in current master branch.

**Comment by SamBrishes** [comment]
Fixed in v1.3.3 (Version has been pushed to Octobers Marketplace).
